src/Pages/AboutPage/About.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Map from './../../assets/Icons/map.png';
+import Adress from './../../assets/Icons/address.png';
+import Phone from './../../assets/Icons/phone.png';
+import Facebook from './../../assets/Icons/facebook.png';
+import Instageam from './../../assets/Icons/instagram.png';
+import Twitter from './../../assets/Icons/twitter.png';
+import './about.scss';
+ 
+export default function About() {
+    return (
+        <main className='about'>
+            <section className='about_text'>
+                <h3 className='about_text__page-title'>Who we are?</h3>
```

Comment:
I'd suggest to add file typography.scss to common styles and put there all styles for each possible text. you can find all typography rules here: https://www.figma.com/file/yHGr3hIT8Tc3FYqjMXyMqS/artSchool?node-id=194%3A38
which text corresponds to what tag you may find on the right sidebar: https://prnt.sc/2076uaj